recipes/neurosynchro/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "neurosynchro" %}
+{% set version = "0.1.3" %}
+{% set sha256 = "54dec70dbb4c3ab2e1226e2e345f5362aa07e64f4f69e648966bb7528d0d96f4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - keras
```

Comment:
are you sure this all needs to be in the host section?

Reply:
As opposed to `build`? My understanding was the `host` was the right place for these kinds of dependencies.

Reply:
I thought the new layout needs to look like this https://github.com/conda-forge/python-coveralls-feedstock/blob/6042143b4a5177551df97628f0c3a3f103a572bb/recipe/meta.yaml#L19
Feel free to ignore me :)

Reply:
I'm not super sure either ... I guess if you're a pure Python package and doing the install with `pip`, you don't *need* to have your runtime deps in the `host` section since the install just copies a bunch of files. Let's give it a try.